params/taiko_config.go

```diff
@@ -55,14 +47,6 @@ var (
 var networkIDToChainConfig = map[*big.Int]*ChainConfig{
 	TaikoMainnetNetworkID:      TaikoChainConfig,
 	TaikoInternalL2ANetworkID:  TaikoChainConfig,
```

Comment:
Since we don't have `InternalL2B` anymore, let's just call the `InternalL2A` as `internalL2`?